core/templates/domain/feedback_message/ThreadMessage.model.ts

```diff
@@ -27,8 +27,9 @@ export interface ThreadMessageBackendDict {
   'message_id': number;
   'text': string;
   // Status and Subject for frontend instances of thread message
-  // domain objects should be null initially.These properties should
-  // only exist when the status or subject changes otherwise non-existent.
+  // domain objects are null and should be supplied if it is first
+  // message of the Thread.These properties should only exist
```

Comment:
`...and are only required to be supplied if the message is the first message of the thread. Otherwise, these properties are only non-null when the status or subject changes, respectively.`

Also one small note that I'm seeing throughout: after every period, have a single space before starting the next sentence.

Reply:
done